controllers/src/test/scala/cromwell/pipeline/controller/AuthControllerTest.scala

```diff
@@ -26,10 +26,12 @@ class AuthControllerTest extends WordSpec with Matchers with MockFactory with Sc
     "signIn" should {
 
       "return token headers if user exists" taggedAs (Controller) in {
-        val signInRequest = SignInRequest("email@cromwell.com", "password")
+        val email = "email@cromwell.com"
+        val password = "password"
         val authResponse = AuthResponse(accessToken, refreshToken, accessTokenExpiration)
-        val httpEntity = HttpEntity(`application/json`, Json.stringify(Json.toJson(signInRequest)))
-        (authService.signIn _ when signInRequest).returns(Future(Option(authResponse)))
+        val signInRequestStr = s"""{"email":"${email}","password":"${password}"}"""
```

Comment:
`email` and `password` could be inlined to the `signInRequestStr`

Reply:
this is used in 34 (used twice) - (in (authService.signIn _ when SignInRequest(email, password)).returns(Future(Option(authResponse))))

Reply:
Ok, agreed.